src/utils/RouteUtils.js

```diff
@@ -56,26 +56,26 @@ async function getRoutes(
   isArriveBy: boolean,
 ): Promise<Array<Object>> {
   const routeResponses = await GhopperUtils.fetchRoutes(end, start, departureTimeQuery, isArriveBy);
+  const { busRoute, walkingRoute } = routeResponses;
 
-  if (!routeResponses) {
-    throw LogUtils.logErr({ message: 'RouteUtils.js: Graphhopper route error : could not fetch routes' });
+  if (!busRoute && !walkingRoute) {
+    LogUtils.log({ message: 'RouteUtils.js: Graphhopper route error : could not fetch routes' });
+    return [];
   }
 
-  const { busRoute } = routeResponses;
-  let { walkingRoute } = routeResponses;
-  // parse the graphhopper walking route=
-  walkingRoute = ParseRouteUtils.parseWalkingRoute(
+  // parse the graphhopper walking route
+  const parsedWalkingRoute = ParseRouteUtils.parseWalkingRoute(
     walkingRoute,
```

Comment:
can't `walkingRoute` be `null` at this point, since it won't be caught by the `if` statement on `61`? Is this acceptable behavior? Same thing regarding just `busRoute`, can't it be `null`? 

Should the `&&` be a `||` statement?

Reply:
it shouldn't be `||` b/c we only want to `return []` if we have no routes at all. i addressed it in a different way